tests/fixtures/issues/issue-092.php

```diff
@@ -13,11 +13,11 @@
             'Version' => '3.9',
```

Comment:
Changes to existing tests are really just to deal with the platform changes I've done, as well as the Nokia device cleanup.